superset/db_engine_specs/base.py

```diff
@@ -878,16 +878,18 @@ def make_label_compatible(cls, label: str) -> Union[str, quoted_name]:
         return label_mutated
 
     @classmethod
-    def get_sqla_column_type(cls, type_: str) -> Optional[TypeEngine]:
+    def get_sqla_column_type(cls, type_: Optional[str]) -> Optional[TypeEngine]:
         """
         Return a sqlalchemy native column type that corresponds to the column type
         defined in the data source (return None to use default type inferred by
-        SQLAlchemy). Override `_column_type_mappings` for specific needs
+        SQLAlchemy). Override `column_type_mappings` for specific needs
         (see MSSQL for example of NCHAR/NVARCHAR handling).
 
         :param type_: Column type returned by inspector
         :return: SqlAlchemy column type
         """
+        if not type_:
```

Comment:
Is this somewhat of an anti-pattern, i.e., passing `type_` as `None` results in  the short-circuit and returning `None` . Can we not call `get_sqla_column_type` if the type is `None`?

Reply:
I'm not sure what calling this with `None` should return. As far as I'm aware, only the Presto db engine spec currently does this, so it would be nice to get more context and tests (see https://github.com/apache/incubator-superset/issues/10743#issuecomment-685040076) to make sure this is working as intended.